docs/guides/ORCHESTRATION_CHAT_COMPLETION.md

```diff
@@ -68,90 +68,81 @@ In addition to the prerequisites above, we assume you have already set up the fo
       ```
 
       </details>
-    
-### Chat completion with Templates
 
-Use a chat completion template to generate a response in German:
+### Create a Client
+
+To use the Orchestration service, create a client and a configuration object:
 
 ```java
-var llmConfig = LLMModuleConfig.create().modelName("gpt-35-turbo").modelParams(Map.of());
+var client = new OrchestrationClient();
 
-var inputParams =
-    Map.of("input", "Reply with 'Orchestration Service is working!' in German");
-var template = ChatMessage.create().role("user").content("{{?input}}");
-var templatingConfig = TemplatingModuleConfig.create().template(template);
+var config = new OrchestrationModuleConfig()
+        .withLlmConfig(LLMModuleConfig.create().modelName("gpt-35-turbo").modelParams(Map.of()));
+```
 
-var config =
-    CompletionPostRequest.create()
-        .orchestrationConfig(
-            OrchestrationConfig.create()
-                .moduleConfigurations(
-                    ModuleConfigs.create()
-                        .llmModuleConfig(llmConfig)
-                        .templatingModuleConfig(templatingConfig)))
-        .inputParams(inputParams);
+Please also refer to [our sample code](../../sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java) for this and all following code examples.
+  
+### Chat Completion
 
-CompletionPostResponse result =
-    new OrchestrationClient().chatCompletion(config);
+Use the Orchestration service to generate a response to a user message:
+
+```java
+var prompt = new OrchestrationPrompt("Hello world! Why is this phrase so famous?");
+
+var result = client.chatCompletion(prompt, config);
 
 String messageResult =
-    result.getOrchestrationResult().getChoices().get(0).getMessage().getContent();
+        result.getOrchestrationResult().getChoices().get(0).getMessage().getContent();
 ```
 
-See [an example in our Spring Boot application](../../sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java)
+In this example, the Orchestration service generates a response to the user message "Hello world! Why is this phrase so famous?".
+The LLM response is available as the first choice under the `result.getOrchestrationResult()` object.
 
-### Message history
+### Chat completion with Templates
 
-Include a message history to maintain context in the conversation:
+Use a prepared template and execute requests with by passing only the input parameters:
 
 ```java
-var llmConfig = LLMModuleConfig.create().modelName("gpt-35-turbo").modelParams(Map.of());
+var template = ChatMessage.create().role("user").content("Reply with 'Orchestration Service is working!' in {{?language}}");
```

Comment:
```suggestion
    var template =
        ChatMessage.create()
            .role("user")
            .content("Reply with 'Orchestration Service is working!' in {{?language}}");
```
Removes the scrollbar on the code sample

Reply:
Fixed 👍🏼